src/library/autodj/autodjfeature.cpp

```diff
@@ -49,7 +49,7 @@ AutoDJFeature::AutoDJFeature(Library* pLibrary,
     }
     qRegisterMetaType<AutoDJProcessor::AutoDJState>("AutoDJState");
     m_pAutoDJProcessor = new AutoDJProcessor(
-            this, m_pConfig, pPlayerManager, m_iAutoDJPlaylistId, m_pTrackCollection);
+        this, m_pConfig, pPlayerManager, m_iAutoDJPlaylistId, m_pTrackCollection);
```

Comment:
Fixed
/me can't wait until we use clang-format so we can stop wasting time on this.